modules/covid_vaccine/app/models/covid_vaccine/v0/expanded_registration_submission.rb

```diff
@@ -49,6 +49,9 @@ class ExpandedRegistrationSubmission < ApplicationRecord
         reg.form_data&.symbolize_keys!
       end
 
+      serialize :form_data, :raw_form_data, JSON
```

Comment:
Should `eligibility_info` also be serialized here?

Reply:
Good catch! YES! Thank you, just pushed up a change!